conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+#Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost/gobblin
```

Comment:
Should be localhost:8020 to be consistent with what's in gobblin-mapreduce.properties.